key-selector.html

```diff
@@ -28,5 +28,5 @@
 <stackable-modal stackable="$ctrl.showAddModal"
   stackable-closed="!err && ($ctrl.selected=result)"
   br-lazy="$ctrl.showAddModal" br-lazy-id="br-key-selector-add-key-modal">
-  <br-generate-key-pair-modal br-identity="$ctrl.identity"></br-generate-key-pair-modal>
+  <br-generate-key-pair-modal br-identity="$ctrl.identity" br-params="$ctrl.params"></br-generate-key-pair-modal>
```

Comment:
@dlongley @lamara put this in as a workaround for the cache issue we have in the key service here: https://github.com/digitalbazaar/bedrock-angular-key/blob/master/key-service.js#L43-L47